modules/network-observability-SRIOV-configuration.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// network_observability/configuring-operator.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-SR-IOV-config_{context}"]
+= Configure monitoring of SR-IOV interface traffic
+In order to collect traffic from a cluster with Single Root I/O Virtualization (SR-IOV) device, the `FlowCollector` `spec.agent.ebpf.privileged` specification value must be `true`. Then, the eBPF agent monitors the host network namespaces. When a pod with virtual functions (VF) interface is created, a new network namespace is created and the VF interface is migrated from the host network namespace to the pod network namespace.
```

Comment:
the need for `privileged` to be able to detect none host netns, today ebpf agent by default watch host netns 

also the VF migrated under certain configuration in sriov policy where we have to specify `IPAM` without no migration will take place, @memodi can share more configs here as needed

Reply:
Discussed with @msherif1234 on Slack and the following updates will be made based on this comment:
- Clarify that egardless to the privileged setting host netns is always visible AND to see other netns we have to set privileged to true
-  make it clear VF interfaces won't always migrate from host netns to pod netns; it only migrates under certain configurations. Add for example if SRIOV policy uses IPAM.

Reply:
Addressed. I added the `SRIOVNetwork` CR IPAM config to the `Prerequisites`. Is that OK @msherif1234 ?